tox.ini

```diff
@@ -74,8 +76,7 @@ commands_pre =
     # freeze environment
     pip freeze
 commands =
-    coverage run -m unittest -v Orange.tests Orange.widgets.tests Orange.canvas.tests
-    coverage run -m unittest discover -v Orange.canvas.tests
```

Comment:
Canvas tests are already discovered in `coverage run -m unittest -v Orange.tests Orange.widgets.tests`, so it is not required to repeat this test.